languages.toml

```diff
@@ -464,21 +464,13 @@ file-types = ["jl"]
 roots = []
 comment-token = "#"
 language-server = { command = "julia", args = [
-        "--startup-file=no",
-        "--history-file=no",
-        "--quiet",
-        "-e",
-        """
-                using LanguageServer;
-                using Pkg;
-                import StaticLint;
-                env_path = dirname(Pkg.Types.Context().env.project_file);
-                server = LanguageServer.LanguageServerInstance(stdin, stdout, env_path, "");
-                server.runlinter = true;
-                run(server);
-        """,
```

Comment:
This seemed to be the recommended config that would properly detect the project dir, as well as set up the linter: https://github.com/julia-vscode/LanguageServer.jl/wiki/Vim-and-Neovim

Reply:
The docs seem to be all over the place though, with README suggesting one version, and kakoune wiki another

Reply:
I think most of the wiki stuff is just outdated and `runserver()` is now the recommended option.

https://github.com/julia-vscode/LanguageServer.jl/blob/master/src/runserver.jl
 `runserver()` should detect an exsting project dir automatically, linting is enabled by default.

At least for me everything seems to work fine with this setup.

Reply:
There seems to be an issue where I get different false positive lint messages for the new and old setup. VScode works fine. Need to dig a bit to see what's going on.

Reply:
Checked it again. Detection of local environments was broken with the old config, now everything should work as expected.

There are some false positives for both configs after the initial loading of a file, but they disappear after an edit.

Reply:
Looks good then 👍🏻